packages/template-retail-react-app/package.json

```diff
@@ -20,6 +20,7 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "npm run compile-translations && npm run compile-translations:pseudo",
     "push": "npm run build && pwa-kit-dev push",
+    "push:ssr-source-map": "cross-env SSR_SOURCE_MAP=true npm run build && pwa-kit-dev push",
```

Comment:
I'm curious about the naming here. Setting `{devtool: 'source-map'}` seems like it would affect more than just `SSR.js`, am I missing something?

Reply:
I am setting `{devtool: 'source-map'}` only within the `ssr` config, which I believe only generates the source map for `ssr.js`. Please lmk if this is incorrect!

Reply:
I think you've got it right, the git diff just omitted the portion that shows that context